packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/aspnet/integratechallengemagiclink.md

```diff
@@ -157,111 +157,32 @@ selectAuthenticatorResponse = await _idxClient.SelectChallengeAuthenticatorAsync
   (selectAuthenticatorOptions, (IIdxContext)Session["IdxContext"]);
 ```
 
-If the call is successful, the returned `selectAuthenticatorResponse` object has an `AuthenticationStatus` of `AwaitingChallengeAuthenticationData`. This indicates that the authenticator can use more than one method to challenge the user and the user must select which one they want to use. These methods are provided in the `selectAuthenticatorResponse.CurrentAuthenticatorEnrollment.MethodTypes` collection and must be presented to the user.
+If the call is successful, OIE sends an email to the user and the returned `selectAuthenticatorResponse` object has an `AuthenticationStatus` of `AwaitingAuthenticatorVerification`. This indicates that OIE is waiting for the user to check their email and either click the magic link or enter the OTP.
```

Comment:
Get we ask Susan and Van about the OIE usage here. I'm not quite sold on it and like to get their take on what to call the Okta code that sends the email.  I ask this because, all our guides should be moving to "OIE" or whatever term is decided to be consistent.

Reply:
@robertnicolo-okta Agreed - we can't keep leaving it at "Okta sends an email" or "Okta does x"

Reply:
Have changed OIE to Identity Engine as discussed elsewhere.